cmd/cli/docs/capact_typeinstance_get.md

```diff
@@ -27,7 +27,8 @@ capact typeinstance get c49b 4793 -oyaml --export > /tmp/typeinstances.yaml
 ```
       --export             Converts TypeInstance to update format.
   -h, --help               help for get
-  -o, --output string      Output format. One of: json | table | yaml (default "table")
+  -o, --output string      Output format. One of: json | jsonpath | table | yaml (default "table")
```

Comment:
I tested it on my end now and in case of:
```
./bin/capact_linux_amd64_v1/capact ti get aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa -ojsonpath
```

It returns nothing. I am not sure what would be the best approach but perhaps just inform the user about the flag `--template`

Reply:
CLI now returns error when output is jsonpath and template is not specified